src/build-disk-image.ts

```diff
@@ -130,7 +130,7 @@ export async function buildDiskImage(imageData: unknown, history: History) {
       // Preliminary Step 0. Create the "bootc-image-builder" container
       // options that we will use to build the image. This will help with debugging
       // as well as making sure we delete the previous build, etc.
-      const buildImageContainer = createBuilderImageOptions(
+      const buildImageContainer = await createBuilderImageOptions(
```

Comment:
IMO we should do something like:

```ts
const containerName = await getUnusedName(buildContainerName)
```

Then pass it into createBuilderImageOptions

Reply:
Yeah, that's better. Fixed in new commit.